vue/router.py

```diff
@@ -2,15 +2,31 @@
 from .factory import Wrapper, VueRouterFactory
 
 
+def init_dict(component):
+    return component.init_dict()
+
+
 class VueRouter(Wrapper):
     @classmethod
     def init_dict(cls):
         return VueRouterFactory.get_item(cls)
 
-    def __new__(cls):
-        return window.VueRouter.new(cls.init_dict())
+    def __new__(cls, VueRouterObject=window.VueRouter):
+        window.Vue.use(VueRouterObject)
```

Comment:
actually not sure about this here.
Maybe this should not be done implicit here but explicit in the application code?

Reply:
I find it convenient even without using a different class.

Reply:
Thanks for changing it!

I agree, it is convenient.
But there are some downsides, which I think are not worth the gain in convenience:
* It introduces a different behavior than vue.js
* It is a side-effect which you do not expect when you do not know the class
* It introduces a new dependency to `window.Vue`